src/test/java/core/basesyntax/HelloWorldTest.java

```diff
@@ -1,8 +1,184 @@
 package core.basesyntax;
 
+import core.basesyntax.model.User;
+import core.basesyntax.service.ImpossibleRegisterUserException;
+import core.basesyntax.service.RegistrationServiceImpl;
+import org.junit.Assert;
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+
+
 /**
  * Feel free to remove this class and create your own.
  */
 public class HelloWorldTest {
+    private RegistrationServiceImpl registrationService = new RegistrationServiceImpl();
+    private User user = new User();
+
+    @BeforeEach
+    void setUp() {
+        user = new User();
+    }
+
+    @Test
+    void loginOfUserAlreadyExists_NotOk() {
+        user.setId(12l);
+        user.setLogin("sanya67");
+        user.setAge(18);
+        user.setPassword("123456");
+        try {
+            Assert.assertEquals(user, registrationService.register(user));
```

Comment:
The use of `Assert.assertEquals` within a try-catch block is not ideal for testing exceptions. Consider using `Assertions.assertThrows` to test for exceptions instead of printing messages in the catch block.